src/components/DogList.css

```
@@ -0,0 +1,6 @@
+.dogListHeader {
+    background-color: lavender;
+    padding: 30px;
+    margin: 20px;
+
+}
```

How about putting the class on the outer div and then using more specific style rules to target the heading?
```suggestion
.dogList>h2 {
    background-color: lavender;
    padding: 30px;
    margin: 20px;
}
```

---

This can be done, thanks!